meinberlin/apps/budgeting/api.py

```diff
@@ -75,6 +78,15 @@ def list(self, request, *args, **kwargs):
                 'choices': label_choices,
             }
 
+        # moderation task filter, only show to moderators
+        if is_allowed_moderate_project(request.user, self.module):
```

Comment:
I first tried with adding a permission to the api, but that gets more complicated in frontend, so I thought it is easiest to simply only add the filter for moderators?

Reply:
Oh yes, I think that's much more elegant than only showing it to moderators in the frontend. :)